app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java

```diff
@@ -134,6 +152,28 @@ private void testAutomatedClaimFullPositive(String collectionId, boolean expecte
     String expectedMessage = String.format("Received Claim for collection Id %s.", collectionId);
     assertEquals(expectedMessage, masResponse.getMessage());
 
+    boolean successOrdering = false;
+    if (expectedExamOrder != null) {
+      if (expectedExamOrder) {
+        log.info("Wait for examOrder code to execute.");
+        for (int pollNumber = 0; pollNumber < 15; ++pollNumber) {
+          Thread.sleep(20000);
+          boolean examOrdered = checkExamOrdered(request.getCollectionId());
+          if (examOrdered) {
+            successOrdering = true;
+            break;
+          } else {
+            log.info(
+                "Exam not ordered yet for collection {}. Waiting and rechecking...",
+                request.getCollectionId());
+          }
+        }
+      } else {
+        log.info("Negative test case for exam ordering TBD via database check. Skipping polling");
+      }
+      assertEquals(successOrdering, expectedExamOrder);
+    }
+
     if (!expectedStatusUpdate) {
```

Comment:
This should not happen for cases which orders exam since we also have pdf generator adnd we want to test that.